lib/job-simulation.rb

```diff
@@ -1,31 +1,58 @@
-require './Stack.rb'
-require './Queue.rb'
+require_relative 'Stack'
+require_relative 'Queue'
 
 class JobSimulation
-  attr_reader :workers, :waiting, :roll
+  attr_reader :employed, :waiting, :waitlist
 
   def initialize (jobs_available, job_seekers)
+    #job_seekers must be larger than jobs_available
+    @waiting = job_seekers - jobs_available
+    give_jobs(jobs_available)
+    create_wait_list
+  end
 
+  def give_jobs(jobs_available)
+    @number = 1
```

Comment:
It's fine to make this variable and use it. But if doing so, it should be a local variable, not an instance variable, as it won't be (shouldn't be) used in other methods as well.